RecoLocalTracker/SiPixelClusterizer/plugins/gpuCalibPixel.h

```diff
@@ -11,25 +11,16 @@
 #include "HeterogeneousCore/CUDAUtilities/interface/cuda_assert.h"
 #include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
 
+// local include(s)
+#include "SiPixelClusterThresholds.h"
+
 namespace gpuCalibPixel {
 
   using gpuClustering::invalidModuleId;
 
-  // calibrationConstants
-  // valid for run2
-  constexpr float VCaltoElectronGain = 47;         // L2-4: 47 +- 4.7
-  constexpr float VCaltoElectronGain_L1 = 50;      // L1:   49.6 +- 2.6
-  constexpr float VCaltoElectronOffset = -60;      // L2-4: -60 +- 130
-  constexpr float VCaltoElectronOffset_L1 = -670;  // L1:   -670 +- 220
-  constexpr int VCalChargeThreshold = 100;
-  //for phase2
-  constexpr float ElectronPerADCGain = 1500;
-  constexpr int8_t Phase2ReadoutMode = 3;
-  constexpr uint16_t Phase2DigiBaseline = 1000;
-  constexpr uint8_t Phase2KinkADC = 8;
```

Comment:
In general, I'm concerned that many of the constants are no longer `constexpr`.
Is there any assessment of the impact this may have on the memory usage and (HLT) reconstruction time ?

Reply:
My intention here was to make these configurable at runtime (within a general refactoring allowing more parameters to come from configuration). While for `phiCuts` I'm confident this is something we will need at a certain point for these I leave @cms-sw/trk-dpg-l2 comment. I may have been too zealous.

Reply:
(at a certain point and for offline HIon for sure)

Reply:
I see... OK, then it makes sense that they are no longer `constexpr`.